ui/apps/platform/cypress/integration/compliance/Compliance.helpers.js

```diff
@@ -103,7 +103,7 @@ export function visitComplianceDashboard() {
 /*
  * Assume location is compliance dashboard.
  */
-export function scanCompliance() {
+function scanCompliance() {
```

Comment:
In every place this was previously used it was called after `visitComplianceDashboard()`, which is identical to the function body of `triggerScan()` below. For consistency only the latter is imported now.